tests/Feature/SettingsTest.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Tests\Feature;
+
+use Carbon\Carbon;
+use Tests\TestCase;
+use App\Alexa\Models\User;
+use App\Alexa\Models\Settings;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class SettingsTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    protected $admin;
+    protected $settings;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->admin = factory(User::class)->states('admin')->create();
+        $this->settings = app('settings');
+    }
+
+    public function testAnAdminCanUpdateSettings()
+    {
+        $projectsSubmissionStart = Carbon::tomorrow();
+        $projectsSubmissionEnd = Carbon::tomorrow()->addDays(1);
+        $minMembersTeam = 1;
+        $maxMembersTeam = 5;
+        $requestData = [
+            'projects_submission_start_at' => $projectsSubmissionStart,
+            'projects_submission_end_at' => $projectsSubmissionEnd,
+            'min_members_team' => $minMembersTeam,
+            'max_members_team' => $maxMembersTeam,
+        ];
+
+        $response = $this->actingAs($this->admin)
+            ->put(route('settings.update'), $requestData);
+
+        $response->assertRedirect();
```

Comment:
If you specify a route to where user is redirected, IMO you should add what is the expected redirect.